rfc-164-retire-email-alert-monitoring-as-a-separate-app.md

```diff
@@ -0,0 +1,76 @@
+# Retire email-alert-monitoring as a separate app
+
+## Summary
+
+Replace the current email-alert-monitoring system with a task built into email-alert-api using the Notify API rather than the current Gmail/App based system, giving us a more stable and testable system and allowing us to retire a repo.
+
+## Problem
+
+email-alert-monitoring is a stand-alone app run as a cronjob (currently on Jenkins, although it's being moved over to EKS). It's designed to confirm that medical and travel alert emails have been sent out on time, and supports an SLA. It's clearly critical that we have some way of knowing that these alerts have gone out, but the current system has a number of problems:
+
+- It relies on a slightly Heath Robinson collection of google accounts
```

Comment:
This is a bit of an exaggeration, AFAICT. The only Google account the monitoring tool uses is `govuk_email_check@digital.cabinet-office.gov.uk`.

The reference to `gov.uk.email@notifications.service.gov.uk` in the README is unnecessary detail, it looks like that's just double-checked as the "FROM" address and no other checks are made.

I think we can delete this line as it doesn't really make the case for a problem.

Reply:
Well, accounts is perhaps an oversimplified blanket term - it's not just the google account, it's the google cloud project that needs to be set up as well (and which there are no instructions for). I admit I've used prejudicial language here, so I'll revise, but trying to replicate this setup has cost a lot of dev and IT time, so I think it is definitely a selling point to get rid of it. (Also, see below for money-savings)